app/Http/Controllers/api/v1/ApplicationController.php

```diff
@@ -75,4 +87,24 @@ public function currentUser()
     {
         return (new UserResource(Auth::user()))->withPermissions();
     }
+
+    /**
+     * Returns the default presentation or 404 if there is no default presentation
+     * or the file is missing.
+     *
+     * @return StreamedResponse
+     */
+    public function defaultPresentation(): StreamedResponse
```

Comment:
Could we somehow use this https://laravel.com/docs/8.x/filesystem#the-public-disk so we don't have to stream the default_presentation via php and can just use the webserver.

Only for the private presentations we have to use the Storage::download to ensure the access is restricted.